ci-operator/config/openshift/release/openshift-release-master__ci-4.8.yaml

```diff
@@ -99,6 +99,13 @@ tests:
       SIZE_VARIANT: compact
       TEST_TYPE: upgrade-conformance
     workflow: openshift-upgrade-azure
+- as: e2e-azure-cilium-compact
+  interval: 48h
+  steps:
+    cluster_profile: azure4
+    env:
+      SIZE_VARIANT: compact
```

Comment:
I'd recommend starting with default `SIZE_VARIANT`. `compact` means 3 masters and 0 workers, so chances are some components might not be able to roll out, causing noise in Cilium jobs

Reply:
Ah, I see